frontend/src/components/common/Form/TextArea/index.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Intent,
+  ITextAreaProps,
+  TextArea as RawTextArea,
+} from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
+import { noop } from "lodash-es";
+import React, { FC, useRef, useState } from "react";
+import { Value } from "../common/constants";
+import { StyledIcon, Wrapper } from "./style";
+
+const DangerIcon = () => {
+  return <StyledIcon icon={IconNames.ISSUE} intent={Intent.DANGER} />;
+};
+
+interface Props extends ITextAreaProps {
+  isValid?: boolean;
+  syncValidation?: Array<(value: string) => boolean>;
+  handleChange: ({ isValid, value, name }: Value) => void;
+}
+
+const TextArea: FC<Props> = (props) => {
+  const [isValid, setIsValid] = useState(true);
+  const inputRef = useRef<HTMLTextAreaElement>(null);
+
+  const { handleChange = noop, ...restProps } = props;
```

Comment:
This noop is temporary, correct? Should we mark it as so?

Reply:
Ah I forgot to make `handleChange` prop optional!

So since `props.handleChange` is optional, I want the default callback function to be `noop`, so I don't have to check if `props.handleChange` is defined before calling it in the block:

```ts
handleChange({ isValid: result, name, value });
```